flopy/utils/cvfdutil.py

```diff
@@ -70,6 +98,10 @@ def segment_face(ivert, ivlist1, ivlist2, vertices):
         list of vertices for cell2.
     vertices : ndarray
         array of x, y vertices
+    normalize : bool
+        normalize the coordinates when looking for hanging nodes. Normalization may be
+        needed in cases where vertices are in real world coordinates and have large
```

Comment:
presumably to avoid precision loss. does one ever want no normalization with large coord values? just wondering if a parameter is needed or if it could be turned on for values above a threshold